add_person.rb

```diff
@@ -0,0 +1,33 @@
+def create_student
+  print 'Age: '
+  age = gets.chomp.to_i
+  print 'Name: '
+  name = gets.chomp
+  print 'Has parent permission? [Y/N]: '
+  permission = gets.chomp.downcase
```

Comment:
- [ ] It would be better if all requests for input and input validation should be placed in the main.rb file and/or classes/methods whose responsibilities are to request or validate input
![image](https://github.com/Kwaku28/OOP-School-Library/assets/46388113/52ef4cef-4966-42fc-a3ec-63f169fbb042)